packages/apex-node/src/tests/constants.ts

```diff
@@ -6,7 +6,8 @@
  */
 
 // Tooling API query char limit is 100,000 after v48; REST API limit for uri + headers is 16,348 bytes
-// local testing shows query char limit to be closer to ~12,300
-export const QUERY_CHAR_LIMIT = 12300;
+// local testing shows query char limit to be closer to ~12,300.
+// Through experimentation, the record limit is around 640 before the REST API limit is hit.
```

Comment:
Do we want to be a bit more conservative on the limit here? Also - is this subject to the character limit, rather than a record limit?

Reply:
I like the idea of being extra conservative too and yes, the API is subject to the character limit rather than record

Reply:
I reduced it down to 500 records.